packages/consumption/src/modules/attributes/AttributesController.ts

```diff
@@ -212,31 +212,34 @@ export class AttributesController extends ConsumptionBaseController {
         await this.attributes.create(localAttribute);
 
         if (
-            localAttribute.content instanceof IdentityAttribute && // nested Local Attributes should only be created for Identity Attributes
+            localAttribute.content instanceof IdentityAttribute && // Local Attributes for children should only be created for Identity Attributes
             localAttribute.content.value instanceof AbstractComplexValue
         ) {
-            await this.createLocalAttributesForNestedAttributeValues(localAttribute);
+            await this.createLocalAttributesForChildrenOfComplexAttribute(localAttribute);
         }
 
         this.eventBus.publish(new AttributeCreatedEvent(this.identity.address.toString(), localAttribute));
 
         return localAttribute;
     }
 
-    private async createLocalAttributesForNestedAttributeValues(localAttribute: LocalAttribute): Promise<void> {
+    private async createLocalAttributesForChildrenOfComplexAttribute(localAttribute: LocalAttribute): Promise<void> {
         if (!(localAttribute.content instanceof IdentityAttribute)) {
             throw new ConsumptionError("Only Identity Attributes are allowed here");
```

Comment:
In another comment it is noted that the majority of error messages uses lowercase "attributes" and also formulations like "identity attribute", although the spelling "Attributes" or "IdentityAttributes" would be preferable. Even if we leave it lowercase like it is for now, I think we should keep the error messages consistent within a file we are currently working with. So perhaps "Identity Attributes" should be changed to "identity attributes" so that it matches the other error messages in the AttributesController